snapcraft/plugins/_ros/wstool.py

```diff
@@ -162,6 +162,23 @@ def _run(self, arguments: List[str]) -> str:
         env = os.environ.copy()
 
         env["PATH"] += ":" + os.path.join(self._wstool_install_path, "usr", "bin")
+
+        if "LD_LIBRARY_PATH" not in env:
+            env["LD_LIBRARY_PATH"] = ""
+        env["LD_LIBRARY_PATH"] += ":" + snapcraft.formatting_utils.combine_paths(
```

Comment:
Not that it matters a great deal, but this will add an unnecessary ":" if LD_LIBRARY_PATH is unset.  

I would encourage the direct use of `":".join(...)` instead of combine_paths...

Reply:
`":".join()` doesn't make this prettier if `LD_LIBRARY_PATH` is unset, you still need to filter out a `None` or an empty string, depending on how it's written. If your concern is the extra colon, however, I've pushed a commit that remedies it.